src/services/ContributingDeveloperAuditService/scm/GitHub/api/GitHubApiClient.ts

```diff
@@ -0,0 +1,218 @@
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+import { soosLogger } from "../../../../../logging/SOOSLogger";
+import { sleep } from "../../../../../utilities";
+
+interface IHttpRequestParameters {
+  baseUri: string;
+  githubPAT: string;
+}
+
+interface IHttpClientParameters extends IHttpRequestParameters {
+  apiClientName: string;
+}
+
+export interface GitHubOrganization {
+  login: string;
+}
+
+export interface GitHubRepository {
```

Comment:
These are partially mapped responses from github, only what's usefull to us, if not we will end up with interfaces with more than 50 props in here.

Reply:
just make a note in the file that the interfaces do not represent the full response